CHANGES.md

```diff
@@ -1,3 +1,21 @@
+## edge-18.11.3
+
+* CLI
+  * **New** `linkerd routes` command displays per-route stats for services with
+    service profiles
+  * **Improved** Add `--open-api` to `linkerd profile` command, to read an
+    OpenAPI / Swagger specification file
+  * **Experimental** New `--ha` flag on `linkerd install`, for HA deployment of
```

Comment:
The rest of our bullet points start with a verb, so for consistency I'd change this to: Add `--ha` flag to `linkerd install` command, ...